### PR TITLE
Add safety stop heading and distance values

### PR DESCRIPTION
**Pull request by mikeodr**
Added to the laser status field the last report of a safety stop of distance and angle reported. If this fails or is unavailable it will just report 0.

### CODE REVIEW

**Comment by mikeodr** [comment]
@mikepurvis @tonybaltovski

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ find_package(catkin REQUIRED COMPONENTS
   diagnostic_updater dynamic_reconfigure laser_proc message_generation
   nodelet rosconsole roscpp sensor_msgs std_msgs std_srvs tf urg_c
 )
+set_directory_properties(PROPERTIES COMPILE_OPTIONS "-std=c++11")
```
Comment (mikeodr) [inline CMakeLists.txt:8]: if this is an issue I can just type `std::vector<foo>::iterator` instead.

Comment (mikepurvis) [reply CMakeLists.txt:8]: Fine with me. :)

File: src/urg_c_wrapper.cpp [hunk 2]
```diff
@@ -406,6 +406,120 @@ bool URGCWrapper::getAR00Status(URGStatus& status)
   return true;
 }
 
+bool URGCWrapper::getDL00Status(UrgDetectionReport& report)
+{
+  // Construct and write DL00 command.
+  std::string str_cmd;
+  str_cmd += 0x02;  // STX
+  str_cmd.append("000EDL005BCB");  // DL00 cmd with length and checksum.
+  str_cmd += 0x03;  // ETX
```
Comment (mikepurvis) [inline src/urg_c_wrapper.cpp:415]: Doesn't have to be now, but if there is any more of this kind of thing, let's move this stuff out to a helper function.

**Comment by mikepurvis** [comment]
LGTM, nice.

**Comment by tonybaltovski** [comment]
LGTM

File: src/urg_c_wrapper.cpp (hunk 2 above)
Comment (mikeodr) [reply src/urg_c_wrapper.cpp:415]: Yeah likely, could have it construct the length, crc and STX/ETX if it comes  in the future.

**Pull request merged**